src/main/java/org/folio/config/DAOConfiguration.java

```diff
@@ -99,4 +109,14 @@ public RolloverErrorDAO rolloverErrorDAO() {
   public RolloverBudgetDAO rolloverBudgetDAO() {
     return new RolloverBudgetDAO();
   }
+
+  @Bean
+  public ExpenseClassDAO expenseClassDAO() {
```

Comment:
do `ExpenseClassDAO` and `BudgetExpenseClassDAO` manipulate different tables in the storage layer?

Reply:
They do. But I couldn't use ExpenseClassDao with existing ExpenseClassService. The service wasn't implemented as Singleton. Refactoring would be considered good form. Can you suggest what would be the best option?

Reply:
As discussed in slack, you can leave it as is.